c2corg_ui/static/js/imageuploader.js

```diff
@@ -278,31 +286,22 @@ app.ImageUploaderController.prototype.save = function() {
   var defer = this.q_.defer();
 
   this.api_.createImages(this.files, this.documentService.document)
-  .then(function() {
+  .then(function(data) {
     var meta;
     var id;
+    var images = data['config']['data']['images'];
 
     $('.img-container').each(function(i, image) {
-      meta = this.files[i]['metadata'];
+      meta = images[i]['metadata'];
```

Comment:
There's probably a problem with that change, or at least with this piece of code.

When uploading an image and pressing the "Save" button, the console warns me that
> angular.js:13708 TypeError: Cannot set property 'document_id' of undefined
    at .<anonymous> (imageuploader.js:304)

=> ``meta`` is undefined. It seems there is no ``metadata`` property in ``images[i]``.
Perhaps the correct code would rather be
```
meta = images[i]
```
?
At least it seems to work in my instance.